packages/bigquery/package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "dependencies": {
     "@google-cloud/bigquery": "^5.12.0",
-    "@openfn/language-common": "workspace:1.10.3",
+    "@openfn/language-common": "1.11.1",
```

Comment:
I this case bigquery is locked to a specific version number. I don't know why (maybe we can do some github archaeology and work it out). 

This PR is updating the dependency to the next minor version. That's _probably_ fine but I wouldn't be comfortable doing it without checking first.

Reply:
@josephjclark i am getting this error when i run `pnpm test`  if i use the workspace version 
```
> @openfn/language-bigquery@2.0.2 test /Users/openfn/Workspace/adaptors/packages/bigquery
> mocha --experimental-specifier-resolution=node --no-warnings


file:///Users/openfn/Workspace/adaptors/packages/bigquery/src/Adaptor.js:130
  http,
  ^^^^
SyntaxError: The requested module '@openfn/language-common' does not provide an export named 'http'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)
```

Reply:
That's why i locked the `common` version to `1.11.1` which is the latest version. Should i perhaps use `^1.10.3`

Reply:
That's actually quite an interesting problem. The workspace dependency, the actual adjacent source code, is no longer compatible with some modules.

We haven't yet bumped common and http to 2.0 have we? If we bump  them, we can make sure the dependencies are of the form `^1.10.3`. I don't think we need `workspace:` in front.

I wonder what `workspace:^1...` even means. Will it build a dependency from a tagged source branch? That' some serious muscle under the hood. I suspect we don't want `workspace:` and, strange as it seems, modules with legacy dependencies should download common from npm